src/controllers/account/index.ts

```diff
@@ -38,8 +63,29 @@ export default class AccountController {
     // if match and hasn't expired, reset the password
     return AccountController.resetPassword(ctx)
   }
+
   public static async resetPassword (ctx: BaseContext) {
     ctx.status = 200
     ctx.body = { ok: true }
   }
+
+  // Endpoint to create hash to send in email to the user
+  public static async register (ctx: BaseContext) {
+    const { email, role } = ctx.request.body
+    ctx.assert(email, 400, Strings.EmailIsRequired)
+    ctx.assert(role, 400, Strings.RoleIsRequired)
+
+    // Generate token, store it
+    const emailToken = await this.generateAndStoreToken(ctx)
+    Methods.sendRegisterEmail(emailToken)
+
+    ctx.status = 200
+    ctx.body = { ok: true }
+  }
+
+  // Endpoint to verify hash from email magic link and send back auth token
+  public static async verifyRegister (ctx: BaseContext) {
```

Comment:
can you add a TODO for whatever this func is supposed to have in it eventually

Reply:
done